src/encoded/static/components/collection.js

```diff
@@ -76,28 +78,57 @@ class Data {
     }
 }
 
-var RowView = function (props) {
-    var row = props.row;
-    var id = row.item['@id'];
-    var tds = row.cells.map( function (cell, index) {
-        if (index === 0) {
-            return (
-                <td key={index}><a href={row.item['@id']}>{cell.value}</a></td>
-            );
+const RowView = (props) => {
```

Comment:
This uses the React 15 stateless component mechanism. The other part of this involves not crashing when objects appear in the schema’s “columns” property. See https://github.com/ENCODE-DCC/encoded/commit/722e95b66bebd956ebec53b1dd34023cb3a93502